che1.js

```diff
@@ -0,0 +1,37 @@
+function generateChessBoard() {
+
+	var $board = document.getElementById('board');
+	var $table = document.createElement('table');
+
+	for (var i = 0; i < 10; i++) {
+		var $row = document.createElement('tr');
+		for (var j = 0; j < 10; j++) {
+			var $cell = document.createElement('td');
+
+			var num = '';
+			var str = 'HGFEDCBA';
+
+			if (i == 0 || i == 9) {
```

Comment:
рекомендую использовать строгое сравнение
```suggestion
			if (i === 0 || i === 9) {
```